Source/DafnyLanguageServer.Test/Util/ClientBasedLanguageServerTest.cs

```diff
@@ -97,6 +97,10 @@ public class ClientBasedLanguageServerTest : DafnyLanguageServerTestBase, IAsync
       NotificationHandler.For<FileVerificationStatus>(verificationStatusReceiver.NotificationReceived));
   }
 
+  public record Change(Range range, String inserted);
+
+  public record Changes(List<Change> changes);
```

Comment:
Is this one actually being used?

Reply:
Not according to Rider's Find Usages for me - removed and will let CI call us a liar. :)